lms/envs/common.py

```diff
@@ -2759,10 +2932,6 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
 
 DISABLE_ACCOUNT_ACTIVATION_REQUIREMENT_SWITCH = "verify_student_disable_account_activation_requirement"
 
-################ Enable credit eligibility feature ####################
-ENABLE_CREDIT_ELIGIBILITY = True
```

Comment:
This is another case where either the top level setting on the one in the features dict might be referenced.  We should leave both in for now to not cause any unexpected breakages.  Maybe pull this decleration above the decleration of the FEATURES dict to match the layout in `cms/envs/common.py`

Reply:
Ok, I put this setting back in lms/envs/common.py and cms/envs/common.py